src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -256,4 +314,38 @@ describe('Test suite for dictionary concepts components', () => {
     await inputField.instance().handleKeyPress(event, 'malaria');
     expect(spy).toHaveBeenCalled();
   });
+
+  describe('selectInternalMapping', () => {
+    it('should notify a user when they try to search with less than three characters', async () => {
```

Comment:
Where are the less than three characters passed?

Reply:
I somehow mixed up this description. Thanks for the catch.